common/types/admin.go

```diff
@@ -473,6 +473,35 @@ func (v *RestoreDynamicConfigRequest) GetFilters() (o []*DynamicConfigFilter) {
 	return
 }
 
+// AdminDeleteWorkflowRequest is an internal type (TBD...)
+type AdminDeleteWorkflowRequest struct {
+	Domain     string             `json:"domain,omitempty"`
```

Comment:
You should drop json annotations here. AFAICT this is not serialized directly.
We should come with a separate PR to clean those up on other types as well eventually.